core-plugin/build.gradle

```diff
@@ -0,0 +1,15 @@
+dependencies {
+    implementation "io.github.http-builder-ng:http-builder-ng-okhttp:${httpbuilderNgVersion}"
```

Comment:
Using the OkHttp client for the client library. A better optimisation woul dbe to use the client library that is bundled with the server library instead, but I prefer to get something working first and httpbuilder-ng provides exactly that.